lnwallet/wallet.go

```diff
@@ -1536,6 +1539,16 @@ func (l *LightningWallet) handleContributionMsg(req *addContributionMsg) {
 			theirContribution.MultiSigKey.PubKey,
 		)
 
+		// We also bind the channel configurations to send to the signer
+		// so that a validating remote signer can validate commitment
+		// and second-level HTLC transactions.
+		fundingIntent.ConfigurePsbt(
```

Comment:
Shouldn't we also call this for the `*chanfunding.PsbtIntent` above? It seems like we have the same information available. And I believe even for the `*chanfunding.ShimIntent`.

Also, this makes me think that we should probably try to add some sort of integration test for this. So we can test that the derivation info is properly attached in different scenarios (e.g. normal channel funding, PSBT based funding, shim based funding).
Not sure how to best approach this? Maybe have the remote signer write each PSBT it is requested to sign to a specific file (would only happen when being compiled with the `integration` build tag), then verify it in the itest?